cmd/celestia/blob.go

```diff
@@ -0,0 +1,189 @@
+	Short: "Allows to interact with the Blob Service via JSON-RPC",
+	Args:  cobra.NoArgs,
+}
+
+var getCmd = &cobra.Command{
+	Use:   "get [height, namespace, commitment]",
+	Args:  cobra.ExactArgs(3),
+	Short: "Returns the blob for the given namespace by commitment at a particular height.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := rpcClient(cmd.Context())
+		if err != nil {
+			return err
+		}
+
+		height, err := strconv.ParseUint(args[0], 10, 64)
+		if err != nil {
+			return fmt.Errorf("error parsing a height:%v", err)
+		}
+
+		namespace, err := parseV0Namespace(args[1])
+		if err != nil {
+			return fmt.Errorf("error parsing a namespace:%v", err)
+		}
+
+		commitment, err := base64.StdEncoding.DecodeString(args[2])
+		if err != nil {
+			return fmt.Errorf("error parsing a commitment:%v", err)
+		}
+
+		blob, err := client.Blob.Get(cmd.Context(), height, namespace, commitment)
+
+		output, err := prepareOutput(blob, err)
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintln(os.Stdout, string(output))
+		return nil
+	},
+}
+
+var getAllCmd = &cobra.Command{
+	Use:   "get-all [height, namespace]",
+	Args:  cobra.ExactArgs(2),
+	Short: "Returns all blobs for the given namespace at a particular height.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := rpcClient(cmd.Context())
+		if err != nil {
+			return err
+		}
+
+		height, err := strconv.ParseUint(args[0], 10, 64)
+		if err != nil {
+			return fmt.Errorf("error parsing a height:%v", err)
+		}
+
+		namespace, err := parseV0Namespace(args[1])
+		if err != nil {
+			return fmt.Errorf("error parsing a namespace:%v", err)
+		}
+
+		var output []byte
+		blobs, err := client.Blob.GetAll(cmd.Context(), height, []share.Namespace{namespace})
+
+		output, err = prepareOutput(blobs, err)
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintln(os.Stdout, string(output))
+		return nil
+	},
+}
+
+var submitCmd = &cobra.Command{
+	Use:   "submit [namespace, blobData]",
+	Args:  cobra.ExactArgs(2),
+	Short: "Submit the blob at the given namespace. Note: only one blob is allowed to submit through the RPC.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := rpcClient(cmd.Context())
+		if err != nil {
+			return err
+		}
+
+		namespace, err := parseV0Namespace(args[0])
+		if err != nil {
+			return fmt.Errorf("error parsing a namespace:%v", err)
+		}
+
+		parsedBlob, err := blob.NewBlobV0(namespace, []byte(args[1]))
+		if err != nil {
+			return fmt.Errorf("error creating a blob:%v", err)
+		}
+
+		height, err := client.Blob.Submit(cmd.Context(), []*blob.Blob{parsedBlob})
+
+		response := struct {
+			Height     uint64          `json:"uint64"`
+			Commitment blob.Commitment `json:"commitment"`
+		}{
+			Height:     height,
+			Commitment: parsedBlob.Commitment,
+		}
+
+		output, err := prepareOutput(response, err)
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintln(os.Stdout, string(output))
+		return nil
```

Comment:
nit: There is an opportunity to compress code in all cmds by making `printOutput`

Reply:
I'd not want non-blob-related cmds in this PR. Can make it in subsequent. Here https://github.com/celestiaorg/celestia-node/pull/2515 for example

Reply:
I meant all blob cmds